network/network_windows.go

```diff
@@ -302,6 +302,11 @@ func (nm *networkManager) configureHcnNetwork(nwInfo *EndpointInfo, extIf *exter
 		hcnNetwork.Flags = hcn.DisableHostPort
 	}
 
+	if nwInfo.NICType == cns.NodeNetworkInterfaceAccelnetFrontendNIC {
+		hcnNetwork.Type = hcn.Transparent
+		hcnNetwork.Flags = 9216 // temp hardcode until hcsshim supports the flag
```

Comment:
Where do you get this from?

Reply:
we need to set this number with 9216 for accelnet NIC; we need to get this from hcsshim repo but there is no flag on the repo; Suraj will add this to hcsshim repo